doc/source/examples/unresolved-cfd-dem/single-particle-sedimentation/single-particle-sedimentation.rst

```diff
@@ -0,0 +1,362 @@
+
+You can visualize the insertion with Paraview:
+
+.. image:: images/particle_insertion.png
+    :alt: inserted particle at the top of the channel
+    :align: center
+
+The particle has been inserted it is now possible to simulate its sedimentation.
+
+
+-----------------------
+CFD-DEM Parameter File
+-----------------------
+
+The CFD simulation is carried out using the particle inserted within the previous step. We introduce the different sections of the parameter file ``single-particle-sedimentation.prm`` needed to run this simulation. 
+
+Simulation Control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The simulation is run for :math:`2` s with a time step of :math:`0.005` s. The time scheme chosen for the simulation is first order backward difference method (BDF1). The simulation control section is shown:
+
+.. code-block:: text
+
+    subsection simulation control
+      set method           = bdf1
+      set output name      = result_
+      set output frequency = 10
+      set time end         = 2
+      set time step        = 0.005
+      set output path      = ./output/
+    end
+
+
+Physical Properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We set a density of :math:`996.8` kg/m\ :sup:`3` and a kinematic viscosity of :math:`0.0000008379` m\ :sup:`2`/s as to simulate the particle sedimentation in water at :math:`25^\circ\text{C}` (same conditions as in Ferreira et al. [#ferreira2023]_).
+
+
+.. code-block:: text
+
+    subsection physical properties
+      subsection fluid 0
+        set kinematic viscosity = 0.0000008379
+        set density             = 996.7775
+      end
+    end
+
+
+Boundary Conditions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For the boundary conditions, we choose a slip boundary condition on all the walls of the channel.
+
+.. code-block:: text
+
+    subsection boundary conditions
+      set number = 6
+      subsection bc 0
+        set id   = 0
+        set type = slip
+      end
+      subsection bc 1
+        set id   = 1
+        set type = slip
+      end
+      subsection bc 2
+        set id   = 2
+        set type = slip
+      end
+      subsection bc 3
+        set id   = 3
+        set type = slip
+      end
+      subsection bc 4
+        set id   = 4
+        set type = slip
+      end
+      subsection bc 5
+        set id   = 5
+        set type = slip
+      end
+    end
+
+
+Lagrangian Physical Properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This section is identical to the one previously mentioned for the DEM simulation of particle insertion. The only difference is the definition of gravity. For the vertical case, we set :math:`g_y = -9.81` and :math:`g_x = g_z = 0`
+
+
+Void Fraction
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Since we are calculating the void fraction using the particle insertion of the DEM simulation. For this, we need to read the dem files which we already wrote using check-pointing. We, therefore, set the ``read dem`` to ``true`` and specify the prefix of the ``dem file name = dem``.
```

Comment:
```suggestion
Since we are calculating the void fraction using the particle insertion of the DEM simulation, we need to read the dem files which we already wrote using check-pointing. We, therefore, set the ``read dem`` to ``true`` and specify the prefix of the ``dem file name = dem``.
```